src/fixup.js

```diff
@@ -139,4 +140,78 @@
     }
   }
 
+  /* Deprecation warning */
+  var request = new XMLHttpRequest();
+  request.open('GET', '//www.w3.org/TR/tr-outdated-spec.json', true);
+  request.onload = function() {
+    if (request.status >= 200 && request.status < 400) {
+      var outdatedSpecs = JSON.parse(request.responseText);
+      const pathname = window.location.pathname;
+      var spec = Object.keys(outdatedSpecs).filter(function(k) {return pathname.indexOf(k) === 0;}).shift();
+      if (spec) {
+        var css = 'a#deprecationnote:hover{ background-color: transparent }'
+        ,   style = document.createElement('style');
```

Comment:
This should reuse what is already in base.css... or if something is missing, we should add it there.